### ISSUE TITLE
New user: some features not showing in track window?

### ISSUE THREAD

**Issue by jorvis** [igvteam/igv#791]
Totally accepting that this may be my issue, but I'm using IGV and have loaded my genome as well as PASA output (GFF3).  I have an alignment region in the GFF3 like this:

```
contig_364	assembler-sample_mydb_pasa.sqlite	cDNA_match	505890	505946	.	-	.	ID=align_1291321;Target=asmbl_202767 5112 5168 +
contig_364	assembler-sample_mydb_pasa.sqlite	cDNA_match	516354	516357	.	-	.	ID=align_1291321;Target=asmbl_202767 5108 5111 +
contig_364	assembler-sample_mydb_pasa.sqlite	cDNA_match	536017	537251	.	-	.	ID=align_1291321;Target=asmbl_202767 3873 5107 +
contig_364	assembler-sample_mydb_pasa.sqlite	cDNA_match	537699	537970	.	-	.	ID=align_1291321;Target=asmbl_202767 3601 3872 +
contig_364	assembler-sample_mydb_pasa.sqlite	cDNA_match	539362	539429	.	-	.	ID=align_1291321;Target=asmbl_202767 3533 3600 +
...
```

But when I go to position 505,000 on that contig no feature appears there (screenshot attached.)  Curious what I'm missing.

![Screenshot from 2020-04-09 16-21-21](https://user-images.githubusercontent.com/330899/78942018-652ae780-7a7e-11ea-97ca-96152003de95.png)

**Comment by jrobinso** [comment]
Looking into it.  Note that all those features have the same ID.   I'm not sure how IGV treats that, but its probably the root of the issue.  I suppose they should all be joined as a single feature.

**Comment by jrobinso** [comment]
Well, congratulations, for a new user you are doing well.   This is the first report of this issue.  The problem is IGV is not correctly treating multi-line GFF features such as you have.  I have fixed this in the web version, the desktop IGV fix will take a little longer.   In the end this is what your features (actually its a single feature spread across 5 lines) should look like (from the web version)

<img width="687" alt="Screen Shot 2020-04-09 at 10 50 31 PM" src="https://user-images.githubusercontent.com/933148/78966623-2cf2cb80-7ab5-11ea-878e-40f54a373ee2.png">

**Comment by jorvis** [comment]
Thanks for looking into this, and at such a late hour!  For what it's worth, this is native output from [PASA](https://github.com/PASApipeline/PASApipeline/wiki).  

And checking, but according to the [GFF3 spec](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md) shared IDs is the correct way to represent these segmented features, right?  Or does the input need to be corrected?

**Comment by jrobinso** [comment]
No the file is correct, this was an igv problem.  Its already fixed, but
not released, if you try the "nightly snapshot" it should work.   There
will be a release next week sometime.

**Comment by jrobinso** [comment]
This fix is too complex for a point release (2.8 branch).   It is fixed in "nightly snapshot" and will be released with the next major update in mid to late summer.
